Makefile

```
@@ -39,6 +45,7 @@ help:
 	@echo "  doc                make HTML docs"
 	@echo "  docserve           run an HTTP server on the docs directory"
 	@echo "  html               to make standalone HTML files"
+	@echo "  htmlview           to open the index page built by the html target in your browser"
```

```suggestion
	@echo "  html               make HTML docs"
	@echo "  htmlview           open the index page built by the html target in your browser"
```
If `doc` and `html` are the same, it would seem clearer to make if they had the same description.